site/source/docs/porting/exceptions.rst

```diff
@@ -4,18 +4,9 @@
 C++ exceptions support
 ==============================
 
-C++ exceptions support can be enabled in emscripten using ``-fexceptions``.
-Note that unlike on most platforms, C++ exceptions are disabled by default due
-to their current overhead, see later.
+By default, exception catching is disabled in Emscripten to avoid overhead.
```

Comment:
I think it might be worth adding a little nuance at the top-line here, since the overhead of wasm EH is quite small. I still think that in the near-term we should keep it disabled because right now the options are:
1. Disabled
2. Use a feature that's not in all/most browsers yet
3. Overhead

It might be worth eventually enabling EH by default once support is widespread.

Reply:
Hm I feel it would make this note unnecessarily verbose. Maybe I'll just remove "to avoid overhead" here? The reasons you outlined become clear to the reader from further explanations under each section below.